app/services/PenaltiesService.scala

```diff
@@ -39,6 +40,15 @@ class PenaltiesService @Inject()(connector: PenaltiesConnector) {
       penalty.period.isDefined && penalty.period.get.submission.submittedDate.isEmpty && !penalty.appealStatus.contains(AppealStatusEnum.Accepted) && !penalty.appealStatus.contains(AppealStatusEnum.Accepted_By_Tribunal))
   }
 
+  def isAnyVATUnpaid(penalties: Option[Seq[LatePaymentPenalty]]): Boolean = {
+    penalties match {
+      case Some(_) => penalties.get.exists(penalty =>
+        penalty.period.paymentStatus != PaymentStatusEnum.Paid
```

Comment:
Just as a precautionary measure - we may want to ensure that the penalty has not been removed due to a successful appeal? Otherwise, it could be that the system shows 'VAT needs to be paid' on penalties that have been appealed successfully